### DISCUSSION TITLE
How to display MimeType alongside file size in LineMode?

### DISCUSSION THREAD

**Discussion by uwla** [sxyazi/yazi#1324] [1. Q&A] [answered]
How to create a Linemode:custom() such that it displays the mimetype (text, directory, jpeg, etc) and the file size (human readable, such as kB, MB, GB) ?

Currently what I have is:

```lua
function Linemode:custom()
  -- local mimetype = self_file.mimetype
  local size = self._file:size()
  size = size and ya.readable_size(size) or ""
  return ui.Line(string.format("%s", size))
  -- return ui.Line(string.format("%s %s", size, mimetype))
end
```

**Comment by sxyazi** [comment, marked as answer]
You can get the mimetype of a file by `mime()`, https://yazi-rs.github.io/docs/plugins/types#app-data.folder-file
